testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -700,8 +830,8 @@ impl BitcoinRegtestController {
     ) -> Option<Transaction> {
         let public_key = signer.get_public_key();
 
-        let output_amt = 2 * DUST_UTXO_LIMIT;
-        let (mut tx, utxos) = self.prepare_tx(&public_key, output_amt, 1)?;
+        let output_amt = 4 * DUST_UTXO_LIMIT;
```

Comment:
Why go from `2` to `4`?  Also, why is `output_amt` necessarily `4 * DUST_UTXO_LIMIT`?

Reply:
Refactored in https://github.com/blockstack/stacks-blockchain/pull/2383/commits/4d576c9ffe82b31c8efe4b4659f30b6fa7e576cd